sources/core/client.cpp

```diff
@@ -288,7 +299,7 @@ client::resend_failed_commands(void) {
   //! dequeue commands and move them to a local variable
   std::queue<command_request> commands = std::move(m_commands);
 
-  while (commands.size() > 0) {
+  while (m_commands.size() > 0) {
```

Comment:
This change would cause a regression since m_commands would be empty after moving it into the local variable

Reply:
If m_commands is empty after the move, then the loop is never entered. And this will always be the case, or am I wrong?

Reply:
You are correct but it would revert the fix introduced in 2a22c32d9b4940e3648cd80b510ee7844368f80c